task/kn-apply/0.1/kn-apply.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: kn-apply
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: cli
+spec:
+  description: >-
+    This task deploys a given image to a Knative Service.
+
+    It uses `kn service apply` to create or update given knative service.
+  params:
+  - name: kn-image
+    description: kn CLI container image to run this task
+    default: gcr.io/knative-releases/knative.dev/client/cmd/kn:latest
+  - name: SERVICE
+    description: Knative service name
+  - name: IMAGE
+    description: Image to deploy
```

Comment:
nit: can we please maintain the consistency in params name?

Reply:
@vinamra28 : now using `KN_IMAGE`

Reply:
Thanks 🙂